.editorconfig

```diff
@@ -0,0 +1,27 @@
+; This file is for unifying the coding style for different editors and IDEs.
```

Comment:
This is very useful but I feel like we should introduce this in another PR.

Reply:
Why another PR? Whats the argument for that? Of course I can delete the file and add it back in another PR but its pointless and causes me just more work, just to get the same thing added back?

Reply:
Splitting things that are unrelated to this work makes everything easier for everyone. If you don't want to operate in a collaborative manner, than you shouldn't expect maintainers do to the same.